src/store/coinPage/coinPageActions.js

```diff
@@ -0,0 +1,25 @@
+import axios from 'axios';
+import {
+  COINPAGE_FETCH_COIN_DATA_PENDING,
+  COINPAGE_FETCH_COIN_DATA_SUCCESS,
+  COINPAGE_FETCH_COIN_DATA_ERROR,
+} from './coinPageReducer';
+
+export const getCoinData = (coin) => async (dispatch, state) => {
+  try {
+    dispatch({
+      type: COINPAGE_FETCH_COIN_DATA_PENDING,
+    });
+    const data = await axios(
+      `${process.env.REACT_APP_API_ENDPOINT}coins/${coin}`
+    );
```

Comment:
https://stackoverflow.com/questions/59769649/recursively-convert-an-object-fields-from-snake-case-to-camelcase